hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -1563,7 +1563,7 @@
         <partition-lost-listeners>
             <partition-lost-listener>com.your-package.YourPartitionLostListener</partition-lost-listener>
         </partition-lost-listeners>
-        <merge-policy>com.hazelcast.cache.merge.LatestAccessCacheMergePolicy</merge-policy>
+        <merge-policy batch-size="100">com.hazelcast.cache.merge.LatestAccessCacheMergePolicy</merge-policy>
```

Comment:
The class `com.hazelcast.cache.merge.LatestAccessCacheMergePolicy` does not exist any more.

Reply:
fixed.